maven-plugin/pom.xml

```diff
@@ -228,6 +228,13 @@
             <artifactId>exec-maven-plugin</artifactId>
             <version>1.5.0</version>
         </dependency>
+
+        <!-- SPA Microfrontends -->
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1.1</version>
+        </dependency>
```

Comment:
Is there a reason for favouring this over the Gson library already included?

Reply:
I didn't realize there was already a JSON library! Fixing.